### PR TITLE
[lab-unsupervised-learning-and-sklearn] Ana Horta

### PR DESCRIPTION
**Pull request by japana26**

### CODE REVIEW

**Comment by ta-data-lis** [comment]
Lab checked OK. Well done! 👍

```
patients['diagnosis_int '] = le.fit_transform(patients.diagnosis.values)
```

The label encoder is assigning a different number to each value that it finds in the `diagnosis` column. This is not quite the desired outcome because it is not grouping the multiple "with diagnosis" values into the same label. Instead, consider using something like
```
patients['diagnosis_int'] = np.where(patients['diagnosis'] == 'no diagnosis', 0, 1)
```
